app/views/editions/import.html.erb

```diff
@@ -0,0 +1,8 @@
+<% breadcrumb :import_edition %>
+
+<div class="page-header">
+	<h1>Import a new game</h1>
+</div>
+<div class="container">
+	<%= render "edition_import_form" %>
```

Comment:
Not sure you need to extract this form. In other ocasions the form is extracted so it can be re-used both in the create and edit actions, but if this will only have a create action, no need for it to be extracted.

Reply:
Done